lib/ts/recipe/emailverification/emailVerifiedClaim.ts

```diff
@@ -0,0 +1,51 @@
+import { SessionClaimValidator, BooleanClaim } from "../session";
+import { RecipeInterface } from "./types";
+
+export class EmailVerifiedClaimClass extends BooleanClaim {
+    constructor(getRecipeImpl: () => RecipeInterface) {
+        super({
+            id: "st-ev",
+            refresh: async (userContext) => {
+                await getRecipeImpl().isEmailVerified({
+                    userContext,
+                });
+            },
+        });
+
+        this.validators = {
+            ...this.validators,
+            isValidated: (minRefetchDelayInSeconds: number = 10, getRedirectPath?: () => string | Promise<string>) => ({
```

Comment:
- Considering renaming `minRefetchDelayInSeconds` variable.
- This SDK does not do any redirection. Only the auth-react SDK does. So maybe, this SDK should take an onFailure: () => Promise<void> callback instead, which is executed when this claim fails. We could also just do nothing if the claim fails and return the array of failed claims, but then from the user's point of view, they will have to check for this in all the places they make API calls and handle it. - To discuss.